src/util/saveArticles.ts

```diff
@@ -134,19 +128,41 @@ async function downloadBulk(listOfArguments: any[], downloader: Downloader): Pro
 async function getAllArticlesToKeep(downloader: Downloader, articleDetailXId: RKVS<ArticleDetail>, dump: Dump, mainPageRenderer: Renderer, articlesRenderer: Renderer) {
   await articleDetailXId.iterateItems(downloader.speed, async (articleKeyValuePairs) => {
     for (const [articleId, articleDetail] of Object.entries(articleKeyValuePairs)) {
+      const nonPaginatedArticleId = articleDetail.title
```

Comment:
Here:
* What is this variable for? I don't understand why we now talk about `nonPaginated`? Is that related to the category handling?
* A `title` is not an `articleId`, here we have a least a problem with variable naming.

Reply:
It seems you have removed it as well, I don't see it. AFAIK there should be nowhere a confusion between "title" and "articleId":
* "Title" is the title of the article, as ZIM metadata and should also be the same as the content of the `<title>` DOM node
* "articleId"  is the identifier of the article, this is typically the one we have in the Mediawiki URL

If there is no proven strong mix of these two concept, then I'm OK to close this discussion. Otherwise a ticket should be made.

Reply:
I've updated getModuleDependencies() in unit tests, so the ambiguity between `title` and `articleId` is no longer present for this method.